relayer/pkg/starknet/starknet.go

```diff
@@ -25,6 +25,9 @@ const (
 	serviceKeyChainlink = "chainlink"
 	ChainName           = "starknet"
 	ChainId             = "devnet"
+	// These are one of the the default addresses based on the seed we pass to devnet which is 123
```

Comment:
These are defined by a specific test env we seed with 123.

They shouldn't be part of our core starknet package of the relayer module, but part of the integration-test module where we should define our env where we test.

Reply:
Adjusted and moved to the setup in the test itself